custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -84,6 +85,49 @@ func (t *Translator) GetRespForPod(response *stackdriver.ListTimeSeriesResponse,
 	return nil, fmt.Errorf("Illegal state")
 }
 
+// GetSDReqForPods returns Stackdriver request for query for multiple pods.
+func (t *Translator) GetSDReqForPods(podList runtime.Object, metricName string) (*stackdriver.ProjectsTimeSeriesListCall, error) {
```

Comment:
And in the other places which are already implemented like `GetSDReqForPod` or `GetRespForPod`.